lib/ReactGridLayout.jsx

```diff
@@ -621,14 +654,68 @@ export default class ReactGridLayout extends React.Component<Props, State> {
         maxH={l.maxH}
         maxW={l.maxW}
         static={l.static}
+        droppingPosition={isDroppingElem ? droppingPosition : undefined}
       >
         {child}
       </GridItem>
     );
   }
 
+  onDragOver = (e: DragOverEvent) => {
+    const { droppingItem } = this.props;
+    const { layout } = this.state;
+    const { layerX, layerY } = e.nativeEvent;
+    const droppingPosition = { x: layerX, y: layerY, e };
+
+    if (!this.state.droppingElem) {
```

Comment:
I don't have a good suggestion here (and sorry this is a bit nit-picky, feel free to ignore it), but is there any way we can rename the `droppingItem` and `droppingElem` so that  the difference is a bit clearer.

Reply:
@n1ghtmare done